.env

```diff
@@ -1,4 +1,7 @@
 BITCOIN_EXPLORER_API_MAINNET=https://mempool.space/api
 BITCOIN_EXPLORER_API_TESTNET=https://mempool.space/testnet/api
 BITCOIN_EXPLORER_API_SIGNET=https://mempool.space/signet/api
+BITCOIN_ELECTRUM_API_MAINNET=pandora.network:50001
+BITCOIN_ELECTRUM_API_TESTNET=pandora.network:60001
+BITCOIN_ELECTRUM_API_SIGNET=pandora.network:60601
```

Comment:
But it's working for you like this?

Reply:
I changed it to use ours, but on the server, please configure it to run with a `esplora.bitmask.app` hostname.